polybar-scripts/wakatime-status/README.md

```diff
@@ -0,0 +1,35 @@
+# Polybar WakaTime status
```

Comment:
Can you rename the script and the directory to `info-wakatime`?
This would fit better in our directory structure.

Reply:
Done.